features/unit/v2algodclient_paths.feature

```diff
@@ -70,16 +70,16 @@ Feature: Algod REST API v2 Paths
       | path                  | application-id |
       | /v2/applications/1234 | 1234           |
 
-  @unit.applications
+  @unit.applications.boxes
   Scenario Outline: GetApplicationBoxByName
     When we make a GetApplicationBoxByName call for applicationID <application-id> with box name "<box-name>"
     Then expect the path used to be "<path>"
     Examples:
-      | path                                     | application-id | box-name     |
-      | /v2/applications/1234/boxes/hello        | 1234           | b64:aGVsbG8= |
-      | /v2/applications/1234/boxes/%FF          | 1234           | b64:/w==     |
-      | /v2/applications/1234/boxes/%F0%9F%92%A9 | 1234           | b64:8J+SqQ== |
-      | /v2/applications/1234/boxes/a%2Fz        | 1234           | b64:YS96     |
+      | path                                              | application-id | box-name     |
+      | /v2/applications/1234/box?name=b64:aGVsbG8%3D     | 1234           | b64:aGVsbG8= |
```

Comment:
@algochoi Minor:  Does _path_ intend represent the values sent on the wire (URL encoded)?

It seems like _yes_.  If so, can you help me understand why it's _safe_ to leave `:` as is (without URL encoding)?

Reply:
> can you help me understand why it's safe to leave : as is (without URL encoding)?

Yes, good point, the Py SDK does not seem to care but it's better to escape `:`

Reply:
@algochoi Thanks for the change - resolving.